src/octoprint/printer/standard.py

```diff
@@ -1222,10 +1222,13 @@ def _payload_for_print_job_event(self, location=None, print_job_file=None, print
 			_, name = self._fileManager.split_path(FileDestinations.LOCAL, path)
 			origin = FileDestinations.LOCAL
 
+		from flask.ext.login import current_user
+
 		result= dict(name=name,
 		             path=path,
 		             origin=origin,
 		             size=print_job_size,
+		             event_user=current_user,
```

Comment:
This doesn't make much sense - `Printer` operates outside of the context of any active flask requests, so attaching users to events triggered internally doesn't make sense.